src/Microsoft.Extensions.Logging.Testing/LoggedTest.cs

```diff
@@ -39,5 +39,7 @@ public IDisposable StartLog(out ILoggerFactory loggerFactory, LogLevel minLogLev
         {
             return AssemblyTestLog.ForAssembly(GetType().GetTypeInfo().Assembly).StartTestLog(TestOutputHelper, GetType().FullName, out loggerFactory, minLogLevel, testName);
         }
+
+        public virtual void AdditionalSetup() { }
```

Comment:
Name it somehow logger related.

Reply:
Well it doesn't have to be logger related setup but I'll name it `AdditionalLoggerSetup`